tests/test_extensions.py

```diff
@@ -92,8 +92,8 @@ def testNestedAbbr(self):
                '*[ABBR]: Abreviation'
```

Comment:
If you are going to change the output, you also need to change the input or the test [fails](https://travis-ci.org/Python-Markdown/markdown/jobs/325987980#L591).